haskell-language-server.cabal

```diff
@@ -283,7 +283,7 @@ common qualifyImportedNames
 -- formatters
 
 common floskell
-  if flag(floskell) && (impl(ghc < 9.2.1) || flag(ignore-plugins-ghc-bounds))
+  if flag(floskell) || flag(ignore-plugins-ghc-bounds))
```

Comment:
trailing paren

Reply:
ugh I did the same error I just corrected in the other pr  🤦‍♂️